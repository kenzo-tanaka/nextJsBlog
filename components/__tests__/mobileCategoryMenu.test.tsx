import React from "react";
import { createRenderer } from "react-test-renderer/shallow";
import MobileCategoryMenu from "../mobileCategoryMenu";

jest.mock("next/router", () => ({
  useRouter() {
    return {
      route: "/",
      pathname: "",
      query: "",
      asPath: "",
    };
  },
}));
const renderer = createRenderer();

test("MobileCategoryMenu", () => {
  renderer.render(<MobileCategoryMenu />);
  const renderedOutput = renderer.getRenderOutput();
  expect(renderedOutput).toMatchSnapshot();
});
