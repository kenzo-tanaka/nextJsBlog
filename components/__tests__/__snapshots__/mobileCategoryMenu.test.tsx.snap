// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MobileCategoryMenu 1`] = `
<div
  className="block sm:hidden grid grid-cols-3 mt-5 mb-8 text-center content-center"
>
  <a
    className="m-auto font-medium text-gray-800 w-full pb-1 "
    href="/"
    onClick={[Function]}
    onMouseEnter={[Function]}
  >
    All
  </a>
  <a
    className="m-auto font-medium text-gray-800 w-full pb-1 "
    href="/categories/dev"
    onClick={[Function]}
    onMouseEnter={[Function]}
  >
    dev
  </a>
  <a
    className="m-auto font-medium text-gray-800 w-full pb-1 "
    href="/categories/work"
    onClick={[Function]}
    onMouseEnter={[Function]}
  >
    work
  </a>
</div>
`;
